bowling/inc/lane.hpp

```diff
@@ -1,8 +1,24 @@
 #pragma once
 
+#include "singleGame.hpp"
+
+#include <iostream>
+#include <string>
+#include <vector>
+
 class Lane {
 private:
+    std::vector<SingleGame> gamesInLine_{};
+    std::string laneName_{};
+
+    void parseFile(std::string fileName);
+    void parseLaneName(std::string laneName);
+
```

Comment:
Please consider moving public members as a first in class. It would be better to see the functions you can use on the class object on the first sight.